src/System.CommandLine.Tests/System.CommandLine.Tests.csproj

```diff
@@ -21,6 +21,7 @@
   <ItemGroup>
     <PackageReference Include="ApprovalTests" Version="5.2.2" />
     <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.5" />
```

Comment:
If we're just adding a dependency for the purpose of the one enum, it would be better just to introduce an enum or use one that's already defined in the BCL.